scripts/sexbound/plugins/pregnant/baby.lua

```diff
@@ -249,39 +249,41 @@ end
 --- Reconciles differences between this actor and the other actor
 function Baby:reconcileEntityGroups(mother, father)
     local geneticTable = self._config.geneticTable or {}
-    local species = mother:getSpecies()
-    local otherSpecies = father:getSpecies()
+    local species = mother:getOffspringSpecies()
```

Comment:
Same as other comment, the lookup in the geneticTable list should remain with the actual base species names for ease of understanding and editing for the normal user. The secondary species name should only be used when an actor's own species is to be returned.

That way people can easily define their own cross-breeding rules, but in case there are none we still have the safety of generating actually spawnable babies for non-supported actors.